Fitness/Controllers/ClassDetailViewController.swift

```diff
@@ -388,14 +390,14 @@ class ClassDetailViewController: UIViewController {
             make.height.equalTo(15)
         }
 
-        tableView.snp.updateConstraints {make in
+        classCollectionView.snp.updateConstraints {make in
             make.left.right.equalToSuperview()
             make.top.equalTo(nextSessionsLabel.snp.bottom).offset(32)
-            make.height.equalTo(tableView.numberOfRows(inSection: 0) * 112)
+            make.height.equalTo(classCollectionView.numberOfItems(inSection: 0) * 112)
         }
 
         var height = 764 + descriptionTextView.frame.height + 111
-        height += CGFloat(tableView.numberOfRows(inSection: 0)*112 + 110)
+        height += CGFloat(classCollectionView.numberOfItems(inSection: 0)*112 + 110)
```

Comment:
we should be converting to AutoLayout

Reply:
Noted, for the purposes of this PR, will open an issue and handle it. For purposes of this PR only, I'll leave this area untouched